frontend/src/entities/configuration/cluster-sizes.tsx

```diff
@@ -0,0 +1,163 @@
+/**
+  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+  Licensed under the Apache License, Version 2.0 (the "License").
+  You may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+import React from 'react';
+import { Button, ExpandableSection, FormField, Input, SpaceBetween } from '@cloudscape-design/components';
+import { ClusterType, IAppConfiguration } from '../../shared/model/app.configuration.model';
+import { FormProps } from '../jobs/form-props';
+import { prefixedSetFields, prefixedTouchFields } from '../../shared/validation';
+import Condition from '../../modules/condition';
+import { InstanceTypeSelector } from '../../shared/metadata/instance-type-dropdown';
+
+
+export type ClusterTypesProps = FormProps<(IAppConfiguration)>;
```

Comment:
Why are there parenthesis around this type?

Reply:
I think we established in [this comment](https://github.com/awslabs/mlspace/pull/95#discussion_r1609204972) that I have an unhealthy obsession with parenthesis.